src/boot/efi/boot.c

```diff
@@ -1576,6 +1576,7 @@ static void config_load_defaults(Config *config, EFI_FILE *root_dir) {
         _cleanup_free_ char *content = NULL;
         UINTN value;
         EFI_STATUS err;
+        EFI_GUID guid = LOADER_GUID;
```

Comment:
Why not always use the MAKE_GUID macro? The compiler should do the right thing, yea? And it's not like guid is reassigned anywhere that would make using a variable good for the logical flow